src/_store/directSelectors.js

```diff
@@ -20,6 +21,12 @@ export const ohlcSelector = state => state.ohlc;
 export const tradingOptionsSelector = state => state.tradingOptions;
 export const statesSelector = state => state.states;
 
+export const tradeParamsSelector = state => state.tradesParams;
+export const tradeProposalInfoSelector = state => state.tradesProposalInfo;
```

Comment:
Let me think about it, originally it is an object containing proposal and a few other stuff, maybe not the case now